src/cast_core/src/CrudActor.cpp

```diff
@@ -881,16 +1049,14 @@ struct CrudActor::PhaseConfig {
     mongocxx::collection collection;
     RetryStrategy::Options options;
     std::vector<std::unique_ptr<BaseOperation>> operations;
-    RetryStrategy strategy;
+    metrics::Operation metrics;
 
     PhaseConfig(PhaseContext& phaseContext,
                 const mongocxx::database& db,
                 ActorContext& actorContext,
                 ActorId id)
         : collection{db[phaseContext.get<std::string>("Collection")]},
-          strategy{actorContext.operation("Crud", id)},
-          options{phaseContext.get<RetryStrategy::Options, false>("RetryStrategy")
-                      .value_or(RetryStrategy::Options{})} {
+          metrics{actorContext.operation("Crud", id)} {
```

Comment:
Should this operation have a longer name to distinguish it from the operations for the individual operations?

Reply:
I would like that but it would be a change in the metrics names and would unnecessarily break history